### ISSUE TITLE
Form Generator don't use FieldType::class

### ISSUE THREAD

**Issue by roromix** [sensiolabs/SensioGeneratorBundle#463]
When generate a Form and the Entity contain a field type in `[date,time,datetime]`, the FormType generated don't use `FieldType::class`.

```php
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ->add('author')
            ->add('content')
            ->add('date', 'datetime')
            ->add('advert')
        ;
    }
```

Since Symfony 2.8

```php
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ->add('author')
            ->add('content')
            ->add('date', 'Symfony\Component\Form\Extension\Core\Type\DateTimeType')
            ->add('advert')
        ;
    }
```

**Comment by maerten** [comment]
Hi! What's the status of this? The generated code doesn't seem t work anymore since Symfony 3.1

I believe 

    $builder->add('mydatefield', 'datetime');

has changed to 

    use Symfony\Component\Form\Extension\Core\Type\DateTimeType;
    // ....
    $builder->add('mydatefield, DateTimeType::class);

And there are probably other fields that changed too.

**Comment by xabbuh** [comment]
@maerten The related pull request (#464) needs to be finished. Maybe you want to take over?

**Comment by HeahDude** [comment]
Fixed by #511.

**Comment by xabbuh** [comment]
@HeahDude You can close here now. :)
